test/integration/test_calls.py

```diff
@@ -0,0 +1,509 @@
+"""
+Integration test for Bandwidth's Voice Voice Calls API
+"""
+
+import os
+import time
+import unittest
+import datetime
+
+
+import bandwidth
+from bandwidth.api import calls_api
+from bandwidth.model.create_call import CreateCall
+from bandwidth.model.create_call_response import CreateCallResponse
+from bandwidth.model.call_direction_enum import CallDirectionEnum
+from bandwidth.model.callback_method_enum import CallbackMethodEnum
+from bandwidth.model.machine_detection_configuration import MachineDetectionConfiguration
+from bandwidth.model.machine_detection_mode_enum import MachineDetectionModeEnum
+from bandwidth.model.call_state_enum import CallStateEnum
+from bandwidth.model.redirect_method_enum import RedirectMethodEnum
+from bandwidth.model.call_state import CallState
+from bandwidth.model.update_call import UpdateCall
+from bandwidth.exceptions import ApiException, UnauthorizedException, ForbiddenException, NotFoundException
+
+try:
```

Comment:
To clarify the comment further - Thinking from OOP perspective we've a Test class where all Test methods are encapsulated and then we've other "one-time" setup code above class definition! This can be either moved to @classmethod(suggested) or externalized to a static utility function lib.